virtdata-lib-basics/src/test/java/io/nosqlbench/virtdata/library/basics/shared/stateful/ShowTest.java

```diff
@@ -23,8 +23,6 @@ public void testBasicStateSupport() {
         assertThat(showFoo.apply(2342343L)).isEqualTo("{foo=23}");
         assertThat(showBar.apply(23423L)).isEqualTo("{bar=Bar}");
         new Clear().apply(234);
-        assertThat(showAll.apply("234").isEmpty());
+        assertThat(showAll.apply("234")).isEqualTo("{}");
```

Comment:
is that the expected output?

Reply:
It is now. This was always a diagnostic function, and using json _object_ formatting is a simplification I think.